### DISCUSSION TITLE
Howto sync the settings?

### DISCUSSION THREAD

**Discussion by goebbe** [AXeL-dev/distract-me-not#105] [Q&A]
I would like to use Syncthing to sync the settings of the Extension between computers. 
Does anybody have an idea in which file (filename) the settings are stored inside my .mozilla/firefox/profile.name/ folder?

**Comment by AXeL-dev** [comment]
There is already a feature request about settings syncing: https://github.com/AXeL-dev/distract-me-not/issues/58.

Still it's not on the priorities list, so it may take time to be implemented.
